pset-functions.js

```diff
@@ -147,6 +249,39 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 4:')
 
 // Add your code below this line
+function doubleLetters(word) {
+
+  // creating result variable
+  let result
+
+  // making sure we're dealing with a string
+  if (typeof word === "string") {
+    word = word.toLowerCase()
+  } else {
+    return "doubleLetters() requires a string parameter."
+  }
+
+  // double adjacent letter logic
+  for (i = 0; i < (word.length - 1); i++) {
+    if (word[i] === word[1 + i]) {
+      result = true
+    } else {
```

Comment:
Nate, you're crushing these examples; you are simultaneously comparing the array at different indices and taking advantage of where you are in the loop's iteration, really nice job! Small clean up, you can get rid of this else block if it's not doing anything. An else statement is not always required.